conanfile.py

```diff
@@ -266,34 +280,41 @@ def _openssl_path(self):
             pck = "OpenSSL"
         return self.conanfile.deps_cpp_info[pck].rootpath
 
-    def _build(self, dest_dir):
-        from conan.tools.gnu import AutotoolsToolchain, Autotools
-
+    def generate(self, dest_dir):
+        from conan.tools.gnu import AutotoolsToolchain, AutotoolsDeps
         tc = AutotoolsToolchain(self.conanfile, prefix=dest_dir)
         tc.configure_args.extend(["--enable-shared", f"--with-openssl={self._openssl_path}"])
-        tc.make_args.append("-j8")
+        tc.generate()
 
+        tc = AutotoolsDeps(self.conanfile)
```

Comment:
I know that Conan docs reuse the `tc` var name for both `*Toolchain` and `*Deps` but I think it would be nice to assign `*Deps` to a `deps` variable to avoid confusion.